src/plugins/iframe-resizer/client.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = function(client) {
+	window.iframeResizer = {
+		targetOrigin: client._targetOrigin
+	};
+
+	require('iframe-resizer/js/iframeResizer.contentWindow');
```

Comment:
This executes immediately, which is why the `require` is nested within the plugin usage